doc/ref_model/chapters/chapter03.md

```diff
@@ -307,6 +307,36 @@ In a typical overlay network, packets are routed based on networking principles
 
 However, in a service-specific overlay network, packets are routed based on policies. This requires specific support at network level such as  at CNI in CNF environment to provide such specific routing mechanism.
 
+<a name="3.5.1.2"></a>
+#### 3.5.1.2 SFC Architecture
+ The SFC Architecture is built out of architectural building blocks that are logical components to relaize the service function chaining. These consist of management, control and data components. 
```

Comment:
Typo relaize->realize.

Reply:
@sureshkrishnan Hi, i see you have commented on old commit. Can you have look for recent changes as it has been modified from the commit you reviewed as content has been refined . Thanks .